docs/developer-docs/getting-started/cycles/cycles-ledger.mdx

```diff
@@ -0,0 +1,151 @@
+# Cycles ledger
+
+## Overview
+
+When developing and deploying canisters on the Internet Computer, developers need to have **cycles** to pay for the resources used by their canisters. The **cycles ledger** canister has been developed as an alternative to the **cycles wallet** canister for cycles management. 
+
+The biggest difference between the cycles ledger and cycles wallet is that the cycles ledger is a single global ledger canister, whereas the cycles wallet canister must be deployed separately for every developer to be used. Additionally, the cycles wallet has some downsides that the cycles ledger remedies. Some of these are:
+
+- Cycles wallets are a complex solution to associate cycles with principal IDs.
+- Cycles wallets consume cycles themselves, meaning developers need to pay cycles to store cycles.
+- Since the cycles wallet canister must be deployed separately if a cycles wallet canister ID is lost, the cycles in that cycles wallet canister are also lost. 
+
+In short, the cycles ledger simplifies cycle management by providing the ability for principal IDs to hold cycles. 
+
+The cycles ledger currently complies with the ICRC-1, ICRC-2, and ICRC-3 standards. In addition to standard ledger functionality, the cycles ledger also interacts with the NNS canisters and user canisters to provide cycles ledger-specific functions, such as:
+
+- Accepting incoming cycles sent from other canisters.
+- Sending cycles to other canisters.
+- Creating new canisters using cycles.
+
+:::caution
+It is important to note that the cycles ledger does not provide the ability to make arbitrary calls to other canisters with cycles, while the cycles wallet can. This is because open call contexts may cause complications with the cycles ledger. 
+
+If you need to make arbitrary calls to other canisters with cycles, a cycles wallet can be used as before. The cycles ledger can be used to load the cycles wallet with cycles.
+:::
+
+## Using the cycles ledger
+
+### Obtaining ICP
+
+To use the cycles ledger, first, you need ICP tokens. There are a few different ways to obtain ICP tokens, such as:
+
+- Purchasing ICP tokens directly through an exchange that lists ICP tokens available for trade. To see all available places to purchase ICP tokens, check out this [page](https://coinmarketcap.com/currencies/internet-computer/markets/).
+- Trading other tokens for ICP on a decentralized exchange.
+- Receiving tokens as rewards for participating in the governance of the Internet Computer.
+- Receiving a grant of tokens through the Internet Computer Association (ICA) or the DFINITY Foundation.
```

Comment:
Is it necessary/useful to mention grants in this context?

Reply:
It's just standard throughout the docs that when we mention ways to obtain ICP, we include grants as an option.